apps/meteor/app/channel-settings/server/functions/saveRoomName.ts

```diff
@@ -24,10 +30,16 @@ const updateRoomName = async (rid: string, displayName: string, slugifiedRoomNam
 		});
 	}
 
-	return Promise.all([
+	const responses = await Promise.all([
 		Rooms.setNameById(rid, slugifiedRoomName, displayName),
 		Subscriptions.updateNameAndAlertByRoomId(rid, slugifiedRoomName, displayName),
 	]);
+
+	if (responses[1]?.modifiedCount) {
+		void notifyOnSubscriptionChangedByRoomId(rid);
+	}
```

Comment:
A suggestion: we can run the notify attaching a `then` to the returned promise, so you don't have to await and then return
```
return Promise.all([
		Rooms.setNameById(rid, slugifiedRoomName, displayName),
		Subscriptions.updateNameAndAlertByRoomId(rid, slugifiedRoomName, displayName),
	]).then((p) => { void notify....() });

Reply:
hmmm.. isn't mixing both a bit weird? not sure which one looks weirder :P 

```ts
	const responses = await Promise.all([
		Rooms.setNameById(rid, slugifiedRoomName, displayName),
		Subscriptions.updateNameAndAlertByRoomId(rid, slugifiedRoomName, displayName),
	]);

	if (responses[1]?.modifiedCount) {
		void notifyOnSubscriptionChangedByRoomId(rid);
	}

	return responses;
```

vs

```ts
	return Promise.all([
		Rooms.setNameById(rid, slugifiedRoomName, displayName),
		Subscriptions.updateNameAndAlertByRoomId(rid, slugifiedRoomName, displayName),
	]).then((ret) => {
		if (ret[1].modifiedCount) {
			void notifyOnSubscriptionChangedByRoomId(rid);
		}

		return ret;
	});
```

Reply:
honestly, I liked the oplog :trollface: 

Yeah, this is a personal pref so not sure, they're function wise the same so go with the one that looks less ugly :/